airflow/providers/amazon/aws/hooks/s3.py

```diff
@@ -104,11 +106,11 @@ class S3Hook(AwsBaseHook):
         :class:`~airflow.providers.amazon.aws.hooks.base_aws.AwsBaseHook`
     """
 
-    def __init__(self, *args, **kwargs):
-        super().__init__(client_type='s3', *args, **kwargs)
+    def __init__(self, *args: str, **kwargs: str):
```

Comment:
```suggestion
    def __init__(self, *args, **kwargs):
```
We do not add type hints to those (usually it is `tuple` and `dictionary`)

Reply:
Got it, that makes sense. Any ideas on how to get the type hint to work here? Okay to ignore like I did in https://github.com/apache/airflow/pull/10164/commits/573d9a6d73cf565967598b806d411a4a84b6e567?

Reply:
Looking at some other examples, it seems like you can only get the type hinting to work if and when there are other arguments. Also kinda makes me wonder if we need the `def __init__` at all or if we could just call the `super` right away.

Reply:
Going to just revert `573d9a6`.